src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/ResponseNeededNotificationService.java

```diff
@@ -56,21 +57,24 @@ public void sendMail(JobDetail jobDetail) {
                 (String) emailData.get("caseReference")
             );
         } catch (NotificationClientException e) {
-            throw new NotificationException(e);
+            throw new JobExecutionException(e);
         }
     }
 
     @Recover
     public void logNotificationFailure(
-        NotificationException exception,
+        JobExecutionException exception,
         JobDetail jobDetail
-    ) {
+    ) throws JobExecutionException {
         JobDataMap emailData = jobDetail.getJobDataMap();
         logger.info("Failure: failed to send response needed notification ({} to defendant at {}) due to {}",
             emailData.get("caseReference"),
             emailData.get("defendantEmail"),
             exception.getMessage(),
             exception);
+
+        exception.setRefireImmediately(true);
```

Comment:
is refire immediately what we want? when does it stop? how many tries? what happens when it gives up?

Reply:
I am still testing this bit., pushed in to have your view